content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Limites e reservas de ID de processo (PIDs)
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+O Kubernetes permite limitar o número de IDs de processo (PIDs) que um
+{{< glossary_tooltip term_id="pod" text="Pod" >}} pode usar.
+Você também pode reservar um número de PIDs alocáveis para cada {{< glossary_tooltip term_id="node" text="nó" >}}
+para uso pelo sistema operacional e daemons (em vez de Pods).
+
+<!-- body -->
+
+IDs de processo (PIDs) são um recurso fundamental em nós. É trivial alcançar o
+limite de tarefas sem atingir  nenhum outro limite de recurso, o que pode causar
+instabilidade para uma máquina host.
+
+Os administradores de cluster necessitam de mecanismos para garantir que os pods em execução no
+cluster não podem induzir esgostamentos de PID que impedem daemons do host (como o
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} ou o 
+{{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}},
+e potencialmente também o agente de execução de contêiner) da execução.
+Além disso, é importante garantir que os PIDs sejam limitados entre os Pods para
+para garantir que tenham impacto limitado em outras cargas de trabalho no mesmo nó.
+
+{{< note >}}
+Em certas instalações do Linux, o sistema operacional define o limite de PIDs para um padrão 
+como por exemplo `32768`. Considere aumentar o valor de  `/proc/sys/kernel/pid_max`.
+{{< /note >}}
+
+Você pode configurar um kubelet para limitar o número de PIDs que um determinado pod pode consumir.
+Como por exemplo se o sistema operacional host do seu nó estiver configurado para usar um máximo de `262144` PIDs e
+estiver esperando hospedar menos de `250` Pods, pode-se dar a cada Pod uma quota de `1000`
+PIDs para evitar o uso do número geral de PIDs disponíveis desse nó. Se o
+administrador deseja sobrecarregar PIDs de forma semelhante à CPU ou memória, ele também pode fazer isso
+com alguns riscos adicionais. De qualquer forma, um único Pod não será capaz de trazer
```

Comment:
maybe we could have:
"...um único Pod não será capaz de derrubar a máquina".

Reply:
Done